phpstan.neon

```diff
@@ -85,6 +85,61 @@ parameters:
         -
             function: 'dj()'
 
+    ignoreErrors:
+        -
+            message: '#^Cloning object of an unknown class Dom\\HTMLDocument\.$#'
```

Comment:
is message mandatory? 

How I was thinking about it, is something like that: 

```
        -
            identifier: class.notFound
            path: src/core/etl/src/Flow/ETL/Row/Entry/HTMLEntry.php
```

Since this is the only issue we really want to suppress. 

Then you also do the same for related scalar functions and move those suppressions here so after migrating fully to 8.4 there will be just one place to remove them all

Reply:
I understood that you want to exclude as much verbose as possible, but if not, then I adjusted to use a generic exclusion on HTML-related classes